### PR TITLE
UBERF-8889: Fix test suite selection

### LINKED ISSUE #7469: Fix test suite selection in test case selection modal
Fix test suite selection in test case selection modal

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzVjNjAzZmFhYzU4OTk4NTIyNWNjYTAiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIn0.9jDtvpp1KpfvGYeApLoKlRlVDObbXQ8xEfpdC2-_-5o">Huly&reg;: <b>UBERF-8889</b></a></sub>

### PR DESCRIPTION
**Pull request by ArtyomSavchenko**
**Pull Request Requirements:**

- Provide a brief description of the changeset.
- Include a screenshots if applicable
- Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

### CODE REVIEW

File: plugins/test-management-resources/src/components/test-case/TestCasesList.svelte [hunk 1]
```diff
@@ -21,11 +21,11 @@
...
 
-  export let baseQuery: DocumentQuery<Doc> = {}
+  export let query: DocumentQuery<Doc> = {}
   let testCases: number
 
-  const query = createQuery()
-  $: query.query(testManagement.class.TestCase, baseQuery, (res) => {
+  const docQuery = createQuery()
+  $: docQuery.query(testManagement.class.TestCase, query, (res) => {
     testCases = res.length
```
Comment (kristina-fefelova) [inline plugins/test-management-resources/src/components/test-case/TestCasesList.svelte]: Looks like we can use `total` here

```
docQuery.query(testManagement.class.TestCase, query, (res) => {
      testCases = res.total
    },
    { total: true, limit: 1 }
  )
```

File: plugins/workbench-resources/src/components/ComponentNavigator.svelte [hunk 2]
```diff
@@ -53,25 +53,32 @@
...
-        locationQuery = newLocation?.query ?? {}
+        parentQuery = newLocation?.query ?? {}
       })
     )
   }
 
+  function onSelected (e: CustomEvent<any>): void {
+    if (syncWithLocationQuery) return
+    parentQuery = { [parentKey]: e.detail }
+    console.log('it works', resultQuery)
```
Comment (kristina-fefelova) [inline plugins/workbench-resources/src/components/ComponentNavigator.svelte]: remove it?

File: plugins/test-management-resources/src/components/test-case/TestCasesList.svelte (hunk 1 above)
Comment (haiodo) [reply plugins/test-management-resources/src/components/test-case/TestCasesList.svelte]: yep, it is a bad idea to load all testcases just to count them

Comment (ArtyomSavchenko) [reply plugins/test-management-resources/src/components/test-case/TestCasesList.svelte]: Thank you, I was inattentive.
I've created a pull request to fix the same issue in other places: https://github.com/hcengineering/platform/pull/7458

**Review by haiodo — APPROVED** [review]

**Pull request merged**